lms/envs/common.py

```diff
@@ -1291,6 +1295,18 @@
         ],
         'output_filename': 'css/lms-style-course-vendor.css',
     },
+    'style-edx-footer': {
+        'source_filenames': [
+            'sass/footer-v3.css',
```

Comment:
How about just ``footer-edx``? Is there a reason to keep the v-* in the file name?